src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/DecideAnApplicationMidEvent.java

```diff
@@ -91,9 +92,10 @@ private void setSelectedApplicationFields(AsylumCase asylumCase, String code) {
             .findFirst()
             .ifPresent(idValue -> {
                 MakeAnApplication makeAnApplication = idValue.getValue();
+                Optional<List<IdValue<Document>>> mayBeMakeAnApplicationsEvidence = asylumCase.read(MAKE_AN_APPLICATION_EVIDENCE);
```

Comment:
@t0nin0s I would have the correct spelling / camelCase here, should be:

`maybeMakeAnApplicationsEvidence `